apps/docs/content/guides/platform/credits.mdx

```diff
@@ -0,0 +1,60 @@
+---
+id: 'credits'
+title: 'Credits'
+---
+
+## Credit balance
+Each organization has a credit balance. Credits are applied to future invoices to reduce the amount due. As long as the credit balance is greater than $0, credits will be used before charging your payment method on file.
+
+<Image
+  alt="Subscription upgrade modal"
+  src={{
+    light: '/docs/img/guides/platform/credit-balance--light.png',
+    dark: '/docs/img/guides/platform/credit-balance--dark.png',
+  }}
+  zoomable
+/>
+
+You can find the credit balance on the [organization's billing page](https://supabase.com/dashboard/org/_/billing).
+
+### What causes the credit balance to change?
+**Subscription plan downgrades:** Upon subscription downgrade, any prepaid subscription fee will be credited back to your organization for unused time in the billing cycle.\
+As an example, if you start a Pro Plan subscription on January 1 and downgrade to the Free Plan on January 15, your organization will receive approximately $12.50 in credits for the unused time between January 15 and January 31.
```

Comment:
Maybe we should leave out the concrete number here, seems like a place that would be extremely easy to overlook when updating. Or we can calculate it from a centralized value.

Reply:
Good point. Changed it to 50%. This way the reader doesn't have to know that the Pro Plan is currently $25.